modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/components/HighPriorityContacts/SetupHighPriorityContact.jsx

```diff
@@ -54,66 +46,115 @@ const SetupHighPriorityContact = ({
 		return {
 			contactsCategory: {
 				key: _filter,
-				name: filter,
-				role: getContactRoleByFilter(filter),
+				name: filter.toLowerCase(),
+				role: getContactRoleByFilter(filter.toLowerCase()),
 			},
-			filterRequest: new SearchBuilder()
-				.eq('contactsCategory', _filter)
-				.and()
-				.eq(
-					'r_accountEntryToHighPriorityContacts_accountEntryERC',
-					project.accountKey
-				)
-				.build(),
 		};
 	};
 
-	const highPriorityContactsCategory = mapFilterToContactsCategory(filter);
+	useEffect(() => {
+		const result = async () => {
+			try {
+				const data = await getAccountRolesId(project, client);
+				setRolesId(data);
+			} catch (error) {
+				console.error(error);
+			}
+		};
 
-	async function getContacts() {
-		const response = await getHighPriorityContacts(
-			highPriorityContactsCategory.filterRequest
-		);
+		result();
+		// eslint-disable-next-line react-hooks/exhaustive-deps
+	}, [project?.accountKey]);
+
+	const [
+		,
+		{data: userAccountsData},
+	] = useUserAccountsByAccountExternalReferenceCode(project?.accountKey);
+
+	const getHighPriorityContactsByFilterRAYSOURCE = async (filter) => {
```

Comment:
It doesn't seems to be a promise

```suggestion
	const getHighPriorityContactsByFilterRAYSOURCE = (filter) => {
```

also

```suggestion
	const getHighPriorityContactsByFilterRaysource = (filter) => {
```